pages/test_analytics/ruby_collectors.md.erb

```diff
@@ -99,6 +99,20 @@ Once you're done, in your Test Analytics dashboard, you'll see analytics of test
 
 If you don't see branch names, build numbers, or commit hashes in the Test Analytics UI, then see [CI Environments](/docs/test-analytics/ci-environments) to learn more about exporting your environment to the minitest collector.
 
+## Adding annotation spans
+
+This gem allows adding custom annotations to the span data sent to Buildkite using the [.annotate](https://github.com/buildkite/test-collector-ruby/blob/d9fe11341e4aa470e766febee38124b644572360/lib/buildkite/test_collector.rb#L64) method. For example:
```

Comment:
```suggestion
This gem allows adding custom annotations to the span data sent to Buildkite using the `[.annotate](https://github.com/buildkite/test-collector-ruby/blob/d9fe11341e4aa470e766febee38124b644572360/lib/buildkite/test_collector.rb#L64)` method. For example:
```

Suggest to wrap `.annotate` method with `<code>` markup.

Reply:
Secondly, I think the screenshot provided looks different to what's in the app.

The view I'm looking at doesn't have the bounding borders (see screenshot below).

![Screen Shot 2022-10-03 at 11 57 21 am](https://user-images.githubusercontent.com/7202667/193484875-33c3dcf9-a8d4-444b-a347-06228df72bd9.png)

Reply:
oh I think this was taken from the marketing site. Is that right @swebb?

if we need something that'll match the app I guess we can just edit the HTML via developer console and take a screenshot 😄